### ISSUE TITLE
Error when uploading non PDF

### ISSUE THREAD

**Issue by Zigolox** [digitalungdom-se/infrarays.se#60]
Right now it just keeps on spinning.

**Comment by Nautman** [comment]
Are you sure that you are even able to select files that are not a PDF when uploading a recommendation letter?

**Comment by Zigolox** [comment]
Yes when searching I can but this probably depends on operating system/browser. We should have an error message for it either way
![image](https://user-images.githubusercontent.com/41542666/112377849-806e2180-8ce6-11eb-9eac-3f74e0bcb1f1.png)
